app/src/app/[lng]/[inventory]/InventoryResultTab.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { useTranslation } from "@/i18n/client";
+import {
+  InventoryProgressResponse,
+  InventoryResponse,
+  SectorProgress,
+} from "@/util/types";
+import { Box } from "@chakra-ui/react";
+import { TabHeader } from "@/app/[lng]/[inventory]/TabHeader";
+
+export default function InventoryResultTab({
+  lng,
+  inventory,
+  isUserInfoLoading,
+  isInventoryProgressLoading,
+  inventoryProgress,
+}: {
+  lng: string;
+  inventory?: InventoryResponse;
+  isUserInfoLoading?: boolean;
+  isInventoryProgressLoading?: boolean;
+  inventoryProgress?: InventoryProgressResponse;
+}) {
+  const { t } = useTranslation(lng, "dashboard");
+  return (
+    <>
+      {inventory && (
+        <>
```

Comment:
This is not required (it has 1 child)

Reply:
it's required as the child is between curly braces